generators/server/templates/build.gradle.ejs

```diff
@@ -525,8 +527,10 @@ if (SPRING_BOOT_VERSION.indexOf('M') > -1 || SPRING_BOOT_VERSION.indexOf('RC') >
     testImplementation "org.junit.platform:junit-platform-console"
     testImplementation "org.testng:testng"
 <%_ } _%>
-<%_ if (databaseTypeSql) { _%>
+<%_ if (enableLiquibase) { _%>
     liquibaseRuntime sourceSets.main.compileClasspath
+<%_ } _%>
+<%_ if (databaseTypeSql) { _%>
```

Comment:
I omitted on purpose because hibernate is sql and liquibase is mandatory. And liquibase will not be sql only.
If we progress the modularization, dependencies should be injected instead of templates conditions. The injection api is improved at the docker PR, and I didn’t wanted to use the old one.

We can add the condition for completeness.
We will probably need to move cassandra to liquibase to support it with sb3. And there is the neo4j PR migrating to liquibase.